src/simulator.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+import * as vscode from "vscode";
+import { Constants } from "./constants";
+import { IoTHubMessageExplorer } from "./iotHubMessageExplorer";
+import { DeviceItem } from "./Model/DeviceItem";
+import { SimulatorWebview } from "./simulatorwebview/simulatorwebview";
+import { Utility } from "./utility";
+
+export class Simulator {
+
+    public static async getInputDeviceList(): Promise<DeviceItem[]> {
+        return await Utility.getInputDeviceList(Constants.IoTHubAIMessageStartEvent);
```

Comment:
Constants.IoTHubAIMessageStartEvent?

Reply:
Could we use Utility.getFilteredDeviceList ?

Reply:
Resolved.